src/harness/reference_models/ppa/ppa.py

```diff
@@ -141,11 +141,8 @@ def PpaCreationModel(devices, pal_records):
   # after Census Tract Clipping
   contour_union = ops.cascaded_union(device_polygon)
   ppa_polygon = _ClipPpaByCensusTract(contour_union, pal_records)
-
   if ppa_polygon.is_empty:
     raise Exception("Empty Polygon is generated, please check the inputs.")
-  if ppa_polygon.geom_type == "MultiPolygon":
```

Comment:
After further thinking, we are finally OK with that change.

The reference model returns a multipolygon, even though the SAS UUT needs to return a Polygon (for example by connecting the multiple polygons by a thin line contained in the census PAL area). Because the check and spec talks only about being similar within 10% of area then it is fine.
Basically: the complexity of returning one polygon is only on the SAS UUT, and not on the test harness which is not bound by that requirement.

Reply:
Thanks   Serge, we have revert back to original change to to validate the mulipolygon   and raises an exception.     Based on the suggestion we have add one more device in between devices that   are placed inside the service area to pass default test config.